tasmota/xsns_54_ina226.ino

```diff
@@ -124,7 +125,7 @@ static void _debug_fval(const char *str, float fval, uint8_t prec = 4 )
 {
   char fstr[32];
   dtostrfd(fval, prec, fstr);
-  AddLog_P2( LOG_LEVEL_NONE, PSTR("%s: %s"), str, fstr );
+  //AddLog_P2( LOG_LEVEL_NONE, PSTR("%s: %s"), str, fstr );
```

Comment:
This should be enabled but at lest to be used with LOG_LEVEL_DEBUG instead of LOG_LEVEL_NONE

Reply:
Re-enabled and changed per your suggestion.